qiling/loader/pe.py

```
@@ -281,8 +435,8 @@ def call_dll_entrypoint(self, dll: pefile.PE, dll_base: int, dll_len: int, dll_n
         # the blacklist may be revisited from time to time to see if any of the file
         # can be safely unlisted.
         blacklist = {
-            32 : ('gdi32.dll',),
-            64 : ('gdi32.dll',)
+            32 : ('gdi32.dll','user32.dll',),
```

Why blacklist user32.dll?

---

I clarified a little in one of my comments on the PR, but to summarize, user32 DllMain was crashing even before my changes, but Qiling has some code that tries to eat exceptions in DllMain and continue execution anyway. So, examples and tests were still working properly.

However after my changes, user32 DllMain proceeds for longer (since more APIs are working now), but it crashes later in the process in a way that the other recovery code doesn't really handle. So adding it to the blacklist allows x86 to work again. Basically, Qiling was already counting on user32 DllMain to fail, the logic was just somewhere else.

Getting user32 DllMain to work properly would also be a lot of work which is outside the scope/goals of this PR, since it calls a bunch of functions to connect to CSR.